client/reader/stream/index.jsx

```diff
@@ -165,9 +165,8 @@ class ReaderStream extends React.Component {
 		}
 	}
 
-	componentWillReceiveProps( nextProps ) {
-		const { streamKey } = nextProps;
```

Comment:
can't drop this line... :)

Reply:
Good spot! I'll rebase and then fix this up.